x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/index.ts

```diff
@@ -299,9 +301,94 @@ export class APMEventClient {
 
     return this.callAsyncWithDebug({
       operationName,
-      requestType: 'terms_enum',
+      requestType: '_terms_enum',
       params: requestParams,
       cb: (opts) => this.esClient.termsEnum(requestParams, opts),
     });
   }
+
+  async indexTemplate(
+    operationName: string,
+    params: IndicesGetIndexTemplateRequest
+  ) {
+    return this.callAsyncWithDebug({
+      operationName,
+      requestType: '_index_template',
+      params,
+      cb: async (abortOptions) => {
+        try {
+          return await this.esClient.indices.getIndexTemplate(
```

Comment:
I think the `esClient` runs as the current user. Are there cases where the current user won't have access?

Reply:
Yes, the user will need to have all the required permissions. If they don't we should just handle the error gracefully, telling them that they don't have access.